.gitignore

```diff
@@ -11,3 +11,6 @@ doc/api/
 # Files with private data. Used for development and testing
 lib/src/assets/config.json
 lib/src/assets/service_account.json
+
+# IDE
+.idea/
```

Comment:
Please, no. These should be ignored globally.

Reply:
Oups sorry. Will revert

Reply:
reverted